test/unit/test_tables.py

```diff
@@ -1,62 +1,127 @@
 import pytest
 import mock
-from pretty_tables import PrettyTables
-
-
-TABLE = None
-HEADERS = ['ID', 'Name', 'Occupation', 'Employed']
-ROWS = [
-    [1, 'Justin', 'Software Engineer', True],
-    [2, 'Misty', 'Receptionist', False],
-    [3, 'John', None, False],
-]
+from pretty_tables import PrettyTables, TableColors
+
+
+def test_generate_table(headers, rows):
+    pretty_table = PrettyTables.generate_table(headers, rows, 'No data')
+    assert pretty_table == "| ID | Name   | Occupation        | Employed |\n" \
+                           "| 1  | Justin | Software Engineer | True     |\n" \
+                           "| 2  | Misty  | Receptionist      | False    |\n" \
+                           "| 3  | John   | No data           | False    |"
+
+
+def test_generate_table_with_colors(headers, rows, colors):
+    pretty_table = PrettyTables.generate_table(headers, rows, 'No data', colors)
+    assert pretty_table == \
+           '| \x1b[94mID\x1b[0m | \x1b[95mName  \x1b[0m | \x1b[1mOccupation       \x1b[0m ' \
+           '| \x1b[92mEmployed\x1b[0m |\n' \
+           '| \x1b[94m1 \x1b[0m | \x1b[95mJustin\x1b[0m | \x1b[1mSoftware Engineer\x1b[0m ' \
+           '| \x1b[92mTrue    \x1b[0m |\n' \
+           '| \x1b[94m2 \x1b[0m | \x1b[95mMisty \x1b[0m | \x1b[1mReceptionist     \x1b[0m ' \
+           '| \x1b[92mFalse   \x1b[0m |\n' \
+           '| \x1b[94m3 \x1b[0m | \x1b[95mJohn  \x1b[0m | \x1b[1mNo data          \x1b[0m ' \
+           '| \x1b[92mFalse   \x1b[0m |'
```

Comment:
Would it be just as easy to remove the `\` breaks from each line and wrap this in parentheses? This goes for all these tests.

eg:
```python
assert pretty_table == (
    '| \x1b[94mID\x1b[0m | \x1b[95mName  \x1b[0m | \x1b[1mOccupation       \x1b[0m '
    '| \x1b[92mEmployed\x1b[0m |\n'
    '| \x1b[94m1 \x1b[0m | \x1b[95mJustin\x1b[0m | \x1b[1mSoftware Engineer\x1b[0m '
    '| \x1b[92mTrue    \x1b[0m |\n'
    '| \x1b[94m2 \x1b[0m | \x1b[95mMisty \x1b[0m | \x1b[1mReceptionist     \x1b[0m '
    '| \x1b[92mFalse   \x1b[0m |\n'
    '| \x1b[94m3 \x1b[0m | \x1b[95mJohn  \x1b[0m | \x1b[1mNo data          \x1b[0m '
    '| \x1b[92mFalse   \x1b[0m |'
)
```

Reply:
Done